docs/source/utilities.rst

```diff
@@ -101,15 +101,9 @@ Model inspection
     :members:
     :member-order: bysource
 
-.. autofunction:: numpyro.contrib.render.get_model_relations
-
 Visualization Utilities
 =======================
 
-render_model
-------------
-.. autofunction:: numpyro.contrib.render.render_model
```

Comment:
Can we keep this entry with the new path?

Reply:
Done. Turned off automodule for `infer.inspect` to avoid duplicate entry.